File: classic puzzle-medium/sum-of-divisors.py
#https://www.codingame.com/training/medium/sum-of-divisors
#sum-of-divisors
#ref:https://www.geeksforgeeks.org/sum-divisors-1-n/
import sys
import math

n = int(input())
s=0
for i in range(1,n+1):
    s+=(n//i)*i
print(s)

File: classic puzzle-medium&above/sum-of-divisors.js
//https://www.codingame.com/training/medium/sum-of-divisors
//sum-of-divisors
//ref:https://www.geeksforgeeks.org/sum-divisors-1-n/
const n = parseInt(readline());
var s=0;
for(let i=1;i<=n;i++)
    s+=Math.floor(n/i)*i;
console.log(s);
